### PR TITLE
47 about section

### PR DESCRIPTION
**Pull request by Vansh-Baghel**
* Completed about section.
* Responsive.

### CODE REVIEW

File: src/pages/About.js
Comment (JayeshVP24) [inline src/pages/About.js:1]: seems like u did not understand your task.
u were required to create a component in index page.
no where it was mentioned to create a new page!

do make these changes

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Comment by riyavishwa981** [comment]
You don't have to create a separate page in pages folder make a file in components and import it in index.js inside pages

**Comment by Vansh-Baghel** [comment]
> You don't have to create a separate page in pages folder make a file in components and import it in index.js inside pages

I moved it in the components folder in my recent issue, want me to import it in the index js as well?

**Comment by riyavishwa981** [comment]
> > You don't have to create a separate page in pages folder make a file in components and import it in index.js inside pages
> 
> I moved it in the components folder in my recent issue, want me to import it in the index js as well?

Yes you have to add it in index.js which is present inside pages folder so that we can view the changes you have made

**Comment by Vansh-Baghel** [comment]
> components

Done

**Comment by JayeshVP24** [comment]
u forgot `About` badge. check design. above title.
u forgot `+` after numbers in countup
left align everthing on mobile, center align isn't looking as good

**Comment by JayeshVP24** [comment]
you really should read the changes we make in your PRs. 
Its really important for you that you learn from your mistakes.
Your code quality has not changed yet. 
I highly suggest you to read the changes we do with focus and understand them.

**Review by JayeshVP24 — APPROVED** [review]
LGTM

File: src/components/HomepageFeatures/About.js [hunk 1]
```diff
@@ -31,26 +31,26 @@ const About = () => {
               competition is the order of the day.
             </div>
             <div className='flex items-start justify-between w-full'>
-              <div className='flex flex-col items-start p-3 w-[45%] gap-2 bg-[#302261] rounded-xl'>
+              <div className='flex flex-col items-start p-3 w-[45%] gap-2 border border-[#60548A] bg-[#302261] rounded-xl'>
```
Comment (Vansh-Baghel) [inline src/components/HomepageFeatures/About.js]: It looks different than the design in figma 🤔

Comment (JayeshVP24) [reply src/components/HomepageFeatures/About.js]: nope. look closely. border exists

**Pull request merged**